environment-lock.yml

```diff
@@ -1,7 +1,7 @@
 name: slackbotr
 channels:
   - conda-forge
-  - nodefaults
+  - defaults
 dependencies:
```

Comment:
This should be `nodefaults` instead. Can you update that please? I'll update the script separately to ensure that line is correct.

Reply:
Will do

Reply:
done